<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorWindowClass</class>
 <widget class="QMainWindow" name="DoctorWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoctorWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Patients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="patientlistWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>New diagnosis</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="updateDiagnosislineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>New Spec</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="updateSpecialisationlineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="updatepushButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="patientcheckBox">
      <property name="text">
       <string>Show current patients</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="namelineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Diagnosis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="diagnosislineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Specialisation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="specialisationlineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Doctor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="doctorlineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Admission date (dd.mm.yyyy)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="datelineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="addpushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addpushButton</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindowClass</receiver>
   <slot>addPatient()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updatepushButton</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindowClass</receiver>
   <slot>updatePatient()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patientcheckBox</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindowClass</receiver>
   <slot>showCurrentPatients()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>850</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPatient()</slot>
  <slot>updatePatient()</slot>
  <slot>showCurrentPatients()</slot>
 </slots>
</ui>
